src/Behat/Testwork/EventDispatcher/TestworkEventDispatcher.php

```diff
@@ -17,8 +17,27 @@
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
+$isSymfony5AndPhpGreater72 = function () {
```

Comment:
as we're in global scope here I'd rather not create a variable - can you make this self-calling?

```
if ((function(){ ... })()) {
```

Reply:
@ciaranmcnulty me either but I don't think it can be self-calling on php5 which I think you still want to support? Best I could do would probably be to unset the variable immediately after the if block.

Or I could unset it sooner if I moved the logic to a script beside the class definition so this file becomes `if (require(__DIR__.'/_identify_symfony_eventdispatcher_version.php))` and that script cleans up all its variables before it returns.

Reply:
yeah don't worry - I forgot we were supporting such a wide range

Reply:
This is dealt with in the new class_alias version, the var is unset as soon as we've used it so nothing leaks to global state.